sql/patchdemo.sql

```diff
@@ -0,0 +1,28 @@
+CREATE DATABASE IF NOT EXISTS patchdemo;
+
+USE patchdemo;
+
+CREATE TABLE IF NOT EXISTS wikis (
```

Comment:
We do queries with `ORDER BY created`, so there should be an index on it?

Reply:
If you want... but for 100 rows and index won't make much difference